charts/osm/README.md

```diff
@@ -30,7 +30,7 @@ A Helm chart to install the OSM control plane on Kubernetes
 | OpenServiceMesh.prometheus.port | int | `7070` |  |
 | OpenServiceMesh.prometheus.retention.time | string | `"15d"` |  |
 | OpenServiceMesh.replicaCount | int | `1` |  |
-| OpenServiceMesh.serviceCertValidityMinutes | int | `1` |  |
+| OpenServiceMesh.serviceCertValidityDuration | string | `24h` |  |
```

Comment:
Thanks for generalizing the duration and making the default more reasonable!!

A small thing - perhaps not related to this PR - `prometheus.retention.time` is quoted.  Do we have to quote this string duration too?

Reply:
I think this should be quoted like you pointed out. Will follow this PR up with fixing this and the other suggestions.